src/js/index.js

```diff
@@ -0,0 +1,19 @@
+import ScoreController from './controllers/scorecontroller';
+import '../css/style.css';
+
+let scoreController;
+
+function buildDemoView() {
```

Comment:
- [ ] Please, use ES6 syntax on your project, this is a requirement according to good practices on js. 🐶